Makefile

```diff
@@ -10,21 +10,19 @@ endif
 # Currently, we only have F with RV32, and both F and D with RV64.
 ifeq ($(ARCH),RV32)
   SAIL_XLEN := riscv_xlen32.sail
-  SAIL_FLEN := riscv_flen_F.sail
 else ifeq ($(ARCH),RV64)
   SAIL_XLEN := riscv_xlen64.sail
-  SAIL_FLEN := riscv_flen_D.sail
 else
   $(error '$(ARCH)' is not a valid architecture, must be one of: RV32, RV64)
 endif
 
+SAIL_FLEN := riscv_flen_D.sail
```

Comment:
This should perhaps be an option (and hence I haven't deleted the riscv_flen_F.sail file itself), but that's a separate issue. The Makefile needs cleaning up anyway to be more like we have downstream in sail-cheri-riscv where ARCH is not a global parameter but instead per-target so you can have RV32 and RV64 builds coexist (and even do them in parallel with a single make), so I imagine introducing XLEN and FLEN when refactoring to support that would make sense.